bitrise.yml

```diff
@@ -22,11 +22,13 @@ workflows:
     - 1_git_clone_and_post_clone
     - 2_certificate_and_profile
     - 3_provisioning_and_npm_installation
+    - Decision_to_run_UI_Tests
```

Comment:
I don't think moving this here will fix the issue. Let me try to explain why... 
We are running a pipeline with two stages:
1 - configure_build_fennec
2 - build_and_tests (all the UI Tests workflows)

If you move the Decission_to_run_UI_Tests to first stage, as far as I know, ENV Variables are not available for the second stage. So what this solution would do is to not even build the app for the Unit Tests and the second stage will fail since it is expecting the build to run the tests, and those steps are not set to conditionally run based on result from stage 1.

We can discuss this further on slack / zoom. 
More information about Bitrise pipelines: https://devcenter.bitrise.io/en/builds/build-pipelines.html

Reply:
i added the `share_variable` step and also added `Decision_to_run_UI_Tests` to the test themselves for testing purposes. Over the next week or so, if there's a bug still, i wanted to be able to see the output earlier and later, just in case we see an issue of intermediate merges causing false positives in the test run.